common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -757,6 +757,10 @@ def test_badge_share_modal(self):
         profile_page.display_accomplishments()
         badge = profile_page.badges[0]
         badge.display_modal()
+        # In chrome, close button is not inside window
+        # which causes click failures. To avoid this, just change
+        # the position of the popup
+        profile_page.bring_popup_inside_window()
```

Comment:
It would be better if the bring_popup_inside_window method could be added to the Badge object, and called in the close_modal method. 

That way any future uses of the close_modal method (perhaps in additional tests, etc.) will automatically work correctly too.